src/lib/composables/use-align.ts

```diff
@@ -23,15 +23,25 @@ export const UseAlignPropsDefaults: UseAlignProps = {
   align: "top left",
 };
 
+const alignMap = {
+  left: "start",
+  center: "center",
+  right: "end",
+  between: "between",
+  around: "around",
+  evenly: "evenly",
+  // @todo - justify-stretch isn't possible
+  stretch: "stretch",
+} as const;
+
 export default function useAlign(align: UseAlignOptions = "top left") {
-  const alignClass = align
+  const alignments = align
     .split(" ")
-    .map((a) => `${a}-align`)
-    .join(" ");
+    .map((entry) => {
+      const val = alignMap[entry as keyof typeof alignMap];
+      return val ? `justify-${val}` : false;
+    })
+    .filter((entry) => typeof entry === "string");
```

Comment:
`.filter(Boolean)` ?

Reply:
It's an option, but does it really improve readability in regards to what the code does?